### DISCUSSION TITLE
Where does it Save the Data? (Selfhosted/developing)

### DISCUSSION THREAD

**Discussion by acleitao-projects** [toeverything/AFFiNE#8407] [General]
Guys,

I must be really dumb. I'm trying to understand how this software works so I put up a stack on portainer and started to play with.  One thing I cant figure out is where does it save the data we add to the application. I made it run, created few workspaces, added some stuff to it and:
Went to postgres - nothing there besides the workspace and pages the data on the pages couldn't find it
Look for sqllite  db - couldn't find it
Look for json files with the data - nothing there too
MongoDb - couldn't find it lmao

Can some of you developers just point me in the right direction lol?

**Comment by MPapendrecht** [comment]
Hi! I found it underneath ~/.affine/
